### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by lpg-tech**
ChLabOrganization constrains the Organization resource for the ordering and reporting bodies. Parent is CHCoreOrganization.

### CODE REVIEW

File: input/fsh/profiles/Organization.fsh [hunk 1]
```diff
@@ -1,49 +1,7 @@
-/*
-Profile: CHCoreOrganization
+Profile: ChLabOrganization
 Parent: CHCoreOrganization
 Id: ch-lab-organization
 Title: "CH Organization: ordering or reporting Lab-Tests"
```
Comment (ziegm) [inline input/fsh/profiles/Organization.fsh:4]: Title change to "CH LAB Organization: ...."?

**Comment by ziegm** [comment]
@lpg-tech (fyi: @pjolo @oliveregger )
Please do not use the CH LAB Organization everywhere. You can look like EU LAB has it done:
Bundle.entry -> CH Core Organization (already definided by using CH Core Document as parent profile)
Composition.author -> CH Core Organization (dito)
Observation.performer -> CH LAB Organization 
Otherwise we have conflicts with the requirements coming from IPS (tested with CH ELM (broker scenario))

**Review by ziegm — CHANGES_REQUESTED** [review]
see comments

**Comment by lpg-tech** [comment]
Added commit:
https://github.com/hl7ch/ch-lab-report/commit/59b5d7127728adbc5509e3959c3af5eaabf06dd7

**Review by ziegm — APPROVED** [review]
okay for us, thank you

**Pull request merged**